build.gradle.kts

```diff
@@ -5,6 +5,8 @@ val ktlintVersion: String by project
 val githubUser: String by project
 val githubToken: String by project
 
+extra["internalStorageOpenApi"] = "internalStorageOpenApi.json"
```

Comment:
why do we need this _here_? It's not needed for the other files?

Reply:
We use it for client generation in the backend.

Reply:
ok